src/renderer/components/Login/LoginForm.vue

```diff
@@ -106,16 +106,33 @@ export default {
           return false
         }
       })
+    },
+    handleKey (event) {
+      if (!this.selectedInstance) {
+        this.confirm('loginForm')
+      } else {
+        this.login()
+      }
     }
   }
 }
 </script>
 
 <style lang="scss" scoped>
-.login-form {
+.login-form /deep/ {
   margin: 0 auto;
   width: 300px;
 
+  .el-form-item__label {
+    color: #f0f3f9;
+  }
+
+  .el-input__inner {
+    background-color: #373d48;
+    color: #ffffff;
```

Comment:
__[stylelint]__

Expected "#ffffff" to be "#fff" (color-hex-length)


<!-- Comment by [Sider](https://sider.review) -->